src/libs/thorvg/thorvg.h needs to include <cstdint>

### LVGL version

9.2.2

### Platform

SDL on Fedora 34 Linux

### What happened?

Build fails
```
n file included from /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/tvgCommon.h:30,
                 from /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/tvgRender.h:29,
                 from /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/tvgPaint.h:29,
                 from /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/tvgIteratorAccessor.h:29,
                 from /builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/tvgAccessor.cpp:26:
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:188:6: error: elaborated-type-specifier for a scoped enum must not use the â€˜classâ€™ keyword
  188 | enum class BlendMethod : uint8_t
      | ~~~~ ^~~~~
      |      -----
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:188:24: error: found â€˜:â€™ in nested-name-specifier, expected â€˜::â€™
  188 | enum class BlendMethod : uint8_t
      |                        ^
      |                        ::
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:188:12: error: â€˜BlendMethodâ€™ has not been declared; did you mean â€˜RenderMethodâ€™?
  188 | enum class BlendMethod : uint8_t
      |            ^~~~~~~~~~~
      |            RenderMethod
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:189:1: error: expected unqualified-id before â€˜{â€™ token
  189 | {
      | ^
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:351:20: error: â€˜uint8_tâ€™ has not been declared
  351 |     Result opacity(uint8_t o) noexcept;
      |                    ^~~~~~~
/builddir/build/BUILD/lvgl-9.2.2-build/lvgl-9.2.2/src/libs/thorvg/thorvg.h:19:1: note: â€˜uint8_tâ€™ is defined in header â€˜<cstdint>â€™; this is probably fixable by adding â€˜#include <cstdint>â€™
   18 | #include <list>
  +++ |+#include <cstdint>
```

https://download.copr.fedorainfracloud.org/results/fed500/lvgl/fedora-rawhide-x86_64/08812835-lvgl/builder-live.log.gz

### How to reproduce?

_No response_

--- a/src/libs/thorvg/thorvg.h
+++ b/src/libs/thorvg/thorvg.h
@@ -13,6 +13,7 @@
 
 
 #include <functional>
+#include <cstdint>
 #include <memory>
 #include <string>
 #include <list>

